ramls/item.json

```
@@ -39,6 +39,10 @@
       "type": "string",
       "description": "Unique inventory control number for physical resources, used largely for circulation purposes"
     },
+    "shelvingOrder": {
+      "type": "string",
```

As this is a derived property, I think it should be marked read only.

```suggestion
      "type": "string",
      "readonly": true,
```